gobrew version is always dev

```
$ gobrew version
[INFO] gobrew version is dev
```

Yep...

one more:

```
~/.gobrew/bin
 21:03:58  ❯ ./gobrew version
[INFO] gobrew version is dev

~/.gobrew/bin
 21:04:12  ❯ gobrew ls
panic: runtime error: index out of range [2] with length 0

goroutine 1 [running]:
github.com/kevincobain2000/gobrew.(*GoBrew).ListVersions(0xc000137f00)
	/home/runner/work/gobrew/gobrew/gobrew.go:149 +0x8f2
main.main()
	/home/runner/work/gobrew/gobrew/cmd/gobrew/main.go:70 +0x18a
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -19,7 +19,7 @@ jobs:
           version: 'mod'
       -
         name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v4.3.0
+        uses: goreleaser/goreleaser-action@v4
         with:
           distribution: goreleaser
           version: latest

--- a/.goreleaser.yaml
+++ b/.goreleaser.yaml
@@ -4,8 +4,6 @@ before:
 builds:
   -
     main: ./cmd/gobrew
-    ldflags:
-      - -s -w
     env:
       - CGO_ENABLED=0
     goos:

